contrib/examples/deploy.yaml

```diff
@@ -384,6 +388,18 @@ objects:
     groupPriorityMinimum: 10000
     versionPriority: 20
 
+# Create a role for the master installer to save a kubeconfig
+# from a managed cluster
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: "clusteroperator.openshift.io:master-controller"
+  rules:
+  # create secrets with kubeconfig data
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs:     ["create", "update"]
```

Comment:
I would think that you need get/list here as well, since you'd need to update the secret if it already exists (or even delete it, and recreate it)

Reply:
definitely. thanks